main.go

```diff
@@ -216,8 +216,8 @@ func main() {
 					Usage: "Delete all existing clusters (this ignores the --name/-n flag)",
 				},
 				cli.BoolFlag{
-					Name:  "registry-volume",
-					Usage: "Delete the local Docker registry volume",
+					Name:  "force",
+					Usage: "Force the deletion of the cluster",
```

Comment:
We will definitely need some more details on what this does in the helptext and/or in the docs.
`--force` sounds like it would destroy things if they cannot be deleted gracefully, while here it simply disconnects some containers from the network which may not belong to the cluster. :thinking:

Reply:
@iwilltry42 Any suggestion? Maybe we should use a different name for this flag. `--dangling`? `--disconnect`? I would like to keep it a generaic name for the future, so we can _"cleanup"_ other things when drestroying a cluster...

Reply:
I think it's not too much about the naming, but rather about the explanation.
If it's about naming, then what do you think about `--prune`? :thinking:

Reply:
Ok, I will change it to `--prune`.